pom.xml

```diff
@@ -321,9 +325,9 @@
       <version>${quartz.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.simpleframework</groupId>
-      <artifactId>simple-xml</artifactId>
-      <version>${simple-xml.version}</version>
+      <groupId>com.carrotsearch.thirdparty</groupId>
+      <artifactId>simple-xml-safe</artifactId>
```

Comment:
Can't we replace the simple-xml lib with something else? This "safe" variant is from 2019 and has only 9 usages on mvnrepository.com.